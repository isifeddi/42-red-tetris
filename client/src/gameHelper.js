export const S_WIDTH = 12
export const S_HEIGHT = 20

export const Createstage = () => 
Array.from( Array (S_HEIGHT), () => 
    new Array (S_WIDTH).fill([0, 'clear'] ))
