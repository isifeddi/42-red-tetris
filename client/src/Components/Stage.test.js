import renderer from "react-test-renderer";
import Stage from "./Stage";
import { Createstage } from "../gameHelper";

let stage = Createstage(12, 20)

test("Stage render test GameOver ", () => {
  const tree = renderer.create(<Stage gameOver={true}  stage={stage}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
test("Stage render test Stage", () => {
  const tree = renderer.create(<Stage />).toJSON();
  expect(tree).toMatchSnapshot();
});