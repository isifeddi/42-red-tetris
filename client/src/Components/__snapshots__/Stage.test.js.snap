// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stage render test GameOver  1`] = `
<div
  className="sc-gsTCUz geGtPg"
  height={12}
  width={20}
>
  <div
    className="sc-dlfnbm cnNOmY"
  >
    <div
      className="sc-hKgILt eUdMtK"
    >
      <img
        src="README.gif"
        style={
          Object {
            "width": "200px",
          }
        }
      />
      <h1>
        You Won
      </h1>
      <p>
        Wait for host player to restart the game
      </p>
    </div>
  </div>
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
  <div
    className="sc-bdfBwQ jDnFae"
    color="0, 0, 0"
    type={0}
  />
</div>
`;

exports[`Stage render test Stage 1`] = `
<span>
  Loading...
</span>
`;
